scene/resources/shader.cpp

```diff
@@ -176,8 +176,8 @@ bool Shader::is_text_shader() const {
 	return true;
 }
 
-bool Shader::has_uniform(const StringName &p_param) const {
-	return params_cache.has("shader_uniform/" + p_param);
+bool Shader::has_parameter(const StringName &p_name) const {
+	return params_cache.has("shader_uniform/" + p_name);
```

Comment:
Probably should be ``shader_parameter``, but I wonder where the variables are set in the cache